artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,41 @@
+defmodule ArtistQuery do
+  def main(_args) do
+    artist = prompt
+    load_songs("music.csv")
+    |> Enum.filter_map(&(String.upcase(&1["Artist"])) == artist, &(&1["Name"]))
+    |> put_songs(titlecase(artist))
+
+    main([]) # Repeat forever
+  end
+
+  defp put_songs(songs, artist) do
+    message = "* Songs by #{artist} *"
+    separator = String.duplicate("*", String.length(message))
+
+    IO.puts(separator <> "\n" <> message <> "\n" <> separator)
+
+    case songs do
```

Comment:
And, in keeping with a single level of abstraction (let me know if you want more explanation on this), you may want to pull out the displaying of the songs as well (you may need to rename one of the functions).